### DISCUSSION TITLE
Textadept 11.3 beta

### DISCUSSION THREAD

**Discussion by orbitalquark** [orbitalquark/textadept#135] [General]
Hi, Textadept 11.3 beta is released from https://github.com/orbitalquark/textadept/releases/tag/textadept_11.3_beta

Bugfixes:

* Fixed Windows directory typos in the manual.
* Prevent running the command entry while in the command entry.
* Fixed uncommenting comments that are not initially aligned.
* Scintilla: Fixed display of fold lines when wrapped so they are only drawn once per line.
* Scintilla: Fixed crash with too many subexpressions in regex searches.
* Scintilla: Fixed lack of display of underscores in some monospaced fonts on Linux.
* Scintilla: Respond to changes in Linux font scaling.

Changes:

* Updated Makefile lexer to support multiple targets.
* Updated VB lexer to support folding.
* Lexers support more complex folding keywords and improved case-insensitivity.
* Scintilla: Added to `view.element_color` the ability to color fold lines and hidden lines.
* Scintilla: Added [`view.caret_line_highlight_subline`][] to highlight just the subline containing
the caret.
* Scintilla: `view:hide_lines()` can now hide the first line or all lines.
* Scintilla: Make negative settings for extra ascent and descent safer.
* Scintilla: Deprecated `view.property_expanded` in favor of `lexer.property_expanded`.
* Updated to [Scintilla][] 5.1.3.

[`view.caret_line_highlight_subline`]: https://orbitalquark.github.io/textadept/api.html#view.caret_line_highlight_subline
[Scintilla]: https://scintilla.org

**Comment by paaguti** [comment]
Hi,

Apparently there is a certificate problem in the CDK site and I had to ```wget --no-check-certificates -O``` for the CDK sources. Once done, I can compile from scratch
